README.md

```diff
@@ -58,6 +59,20 @@ Using [yarn](https://yarnpkg.com):
 yarn add @fingerprintjs/fingerprintjs-pro-react
 ```
 
+Using [pnpm](https://pnpm.js.org):
+
+```sh
+pnpm add @fingerprintjs/fingerprintjs-pro-react
+```
+
+## Requirements
```

Comment:
Requirements go before "How to install" https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc

Reply:
Done!